src/pretix/eventyay_common/utils.py

```diff
@@ -0,0 +1,60 @@
+import hashlib
+import logging
+import secrets
+from datetime import datetime, timedelta, timezone
+
+import jwt
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+def generate_token(request):
+    """
+    Generate token for video system
+    @param request: user request
+    @return: jwt
+    """
+    uid_token = encode_email(request.user.email)
```

Comment:
Why `email` need to be encoded?

Reply:
I would like to create a unique identification for user authentication in the video system.